corehq/apps/reports/static/reports/js/edit_scheduled_report.js

```diff
@@ -49,6 +52,13 @@ hqDefine("reports/js/edit_scheduled_report", [
                 $('[name="interval"]').change(function () {
                     update_day_input(self.weekly_options, self.monthly_options);
                 });
+                $("#id_start_date").datepicker({
+                    dateFormat: "yy-mm-dd",
+                    minDate: 0,
+                });
+                if (!toggles.toggleEnabled('SET_SCHEDULED_REPORT_START_DATE')) {
```

Comment:
Mildly curious why you're checking the toggle here instead of server side. It's not a big difference, I just lean slightly towards not including the content at all rather than including it and then hiding it.

Reply:
Ah, I didn't know how to remove the `start_date` field in the `Scheduled Report Form` (https://github.com/dimagi/commcare-hq/pull/22775/files#diff-597545574657c656fd164ce865186edaR949)

So, I `disabled` it instead and removed it on the UI side. Is it better to `pop` it from the list of `form.fields`?

Reply:
yep, you can pop it out of `fields`